bundle/Cache/PersistenceCachePurger.php

```diff
@@ -101,12 +101,13 @@ public function resetAllCleared()
      * Either way all location and urlAlias cache is cleared as well.
      *
      * @param int|int[]|null $locationIds Ids of location we need to purge content cache for. Purges all content cache if null
+     * @param int|int[]|null $contentIds Ids of content we need to purge
```

Comment:
could we type hint this as int[]|null? So we can add "array" on method signature? Pr do you see any specific needs to support both?